go/vt/vttablet/tabletserver/tabletserver.go

```diff
@@ -539,6 +525,24 @@ func (tsv *TabletServer) begin(ctx context.Context, target *querypb.Target, save
 	return state, err
 }
 
+func (tsv *TabletServer) getPriorityFromOptions(options *querypb.ExecuteOptions) int {
+	priority := tsv.config.TxThrottlerDefaultPriority
+	if options != nil && options.Priority != "" {
```

Comment:
A small nit: in `vitess` codebase we prefer early returns over indented conditions, so for example we'd do this:
```suggestion
	if options == nil {
	    return priority
	}
	if options.Priority == "" {
	  return priority
	}
```
Mind that if you accept the above suggestion, the code will not compile as we need t remove a closing brace `}`